### PR TITLE
Updated contentent edtor page and its image

### PR DESCRIPTION
**Pull request by seventunes**

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @seventunes,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://285.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://285.qa.docs.sunbird.org/master/developer-docs/how-to-guide/configure_content_metafilter/ <br /> - http://285.qa.docs.sunbird.org/master/developer-docs/how-to-guide/metadata_driven_forms/ <br /> - http://285.qa.docs.sunbird.org/master/features-documentation/contenteditor/ <br /> - http://285.qa.docs.sunbird.org/master/howto/upload_existing_content/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @seventunes,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://285.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://285.qa.docs.sunbird.org/master/developer-docs/how-to-guide/configure_content_metafilter/ <br /> - http://285.qa.docs.sunbird.org/master/developer-docs/how-to-guide/metadata_driven_forms/ <br /> - http://285.qa.docs.sunbird.org/master/features-documentation/contenteditor/ <br /> - http://285.qa.docs.sunbird.org/master/howto/upload_existing_content/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @seventunes,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://285.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://285.qa.docs.sunbird.org/master/developer-docs/how-to-guide/configure_content_metafilter/ <br /> - http://285.qa.docs.sunbird.org/master/developer-docs/how-to-guide/metadata_driven_forms/ <br /> - http://285.qa.docs.sunbird.org/master/features-documentation/contenteditor/ <br /> - http://285.qa.docs.sunbird.org/master/howto/upload_existing_content/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/how-to-guide/configure_content_metafilter.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  </tbody>
+  </table>
+
+## Concepts Covered
+
+<This section should cover in brief, the concepts covered.>
+
+## Related Topics
+
+<This section should provide links to related topics.>
```
Comment (Sunitashelar) [inline developer-docs/how-to-guide/configure_content_metafilter.md]: This file does not follow the structure of a 'How to' doc. Nor are all the sections complete. As a result, this PR cannot be merged. Please do not send PRs for new files along with updated ones. it stalls the merge and publish process.

**Comment by Sunitashelar** [comment]
Cannot merge this PR due to 1 incomplete file. Closing the PR due to this. Please rework and send new PR for other files.

**Pull request closed without merging**